img/o/odoo.com.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 919 495"><path d="M695 346a75 75 0 1 1 75-75 75 75 0 0 1-75 75Zm0-31a44 44 0 1 0-44-44 44 44 0 0 0 44 44Zm-157 31a75 75 0 1 1 75-75 75 75 0 0 1-75 75Zm0-31a44 44 0 1 0-44-44 44 44 0 0 0 44 44Zm-82-45c0 41.9-33.6 76-75 76s-75-34-75-75.9 30.5-74.1 75-74.1c16.4 0 31.6 3.5 44 12.6v-43.5c0-8.3 7.3-15.1 15.5-15.1s15.5 6.8 15.5 15.1Zm-75 45a44 44 0 1 0-44-44 44 44 0 0 0 44 44Z" style="fill:#8f8f8f"/><path d="M224 346a75 75 0 1 1 75-75 75 75 0 0 1-75 75Zm0-31a44 44 0 1 0-44-44 44 44 0 0 0 44 44Z" style="fill:#714b67"/></svg>
```

Comment:
As the image will be constrained to 32x32 pixels, the icon version containing just the "O" would be preferred here.